packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2683,7 +2685,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             // Please note that this does not change file format, so it can be disabled in the future if this
             // optimization no longer makes sense (for example, batch compression may make it less appealing).
             if (this._flushMode === FlushMode.TurnBased && type === ContainerMessageType.Attach &&
-                this.mc.config.getBoolean("Fluid.ContainerRuntime.disableAttachOpReorder") !== true) {
+                this.mc.config.getBoolean("Fluid.ContainerRuntime.enableAttachOpReorder") === true) {
```

Comment:
nit: should we use the same config name so as to not change it later:
`this.mc.config.getBoolean("Fluid.ContainerRuntime.disableAttachOpReorder") === false`

Reply:
Also, this should probably be done is a separate PR to keep its context separate.

Reply:
I started that way but figured there's no hard dependency anywhere on that setting name, and Vlad is going to revisit this code anyway. So I opted for the more natural phrasing.

And sure, I can do a separate PR.